heaps/min_heap.py

```diff
@@ -10,67 +10,97 @@ def __str__(self):
     def __repr__(self):
         return str(self.value)
 
+
 class MinHeap:
 
     def __init__(self):
         self.store = []
 
-
-    def add(self, key, value = None):
+    def add(self, key, value=None):
         """ This method adds a HeapNode instance to the heap
             If value == None the new node's value should be set to key
-            Time Complexity: ?
-            Space Complexity: ?
+            Time Complexity: O(log n)
+            Space Complexity: O(1)
         """
-        pass
+        if value == None:
```

Comment:
👀 Small code styling tip: We generally prefer `is` when comparing to singleton values (None, True, False), since the `==` can be overwritten with `__eq__` and can very occasionally lead to weird behavior
```suggestion
        if value is None:
```